aries-site/src/pages/templates/content-layouts.mdx

```diff
@@ -3,17 +3,55 @@ import { Grid, ResponsiveContext } from 'grommet';
 import { CardGrid, ContentCard } from '../../components';
 import { DiagramPreview, Example } from '../../layouts';
 import { contentLayouts } from '../../data/structures';
-import { PageMarginAnatomy, ResponsiveContentLayoutExample } from '../../examples';
+import { ContainerDrivenLayout, NestedGrid, ResponsiveContentLayoutExample } from '../../examples';
 
-## Designing for Responsiveness
+Content layouts may be applied across different levels of information hierarchy.
+At a page template level, it can help with the layout of page sections. At an "organism" 
+level, it can coordinate how a title, piece of content, and related actions should be 
+laid out in relation to each other.
 
-App layouts should be adaptable to different screen sizes depending on the user input. Our goal as designers and developers at HPE should be to create optimal and accessible user experiences regardless of screen size or resolution.
+Understanding [Grid](/components/grid) is the key to creating successful content layouts.
+
+Looking for a quick start? [Jump to layout templates](#templates).
+
+## Responsive Design
+
+Layouts should be adaptable to different screen sizes depending on the user input. 
+Our goal as designers and developers at HPE should be to create optimal and accessible 
+user experiences regardless of screen size or resolution.
+
+### Breakpoints
+
+A breakpoint is a point at which some change in the page design is likely to occur.
+
+For designers, breakpoints can help stress test and validate UX decisions around page 
+layouts.
+
+See the table below for breakpoints defined by the HPE Design System. Keep in mind that 
+these breakpoints are not meant to target every single existing device &mdash; we understand 
+that technology is ever evolving. Instead, the ranges defined here help build a strong 
+foundation to account for diverse use cases when creating responsive designs.
+
+| Breakpoint     | Range (width in pixels)
+| -------------- | -----------------------
+| **xsmall**     | 576 and below
+| **small**      | 577 - 768
+| **medium**     | 769 - 1080
+| **large**      | 1081 - 1439
+| **xlarge**     | 1440 and above
 
 ### Prioritizing Content Needs
 
-Focusing on content needs can help us create better optimal user experiences–think about which piece of content a user is likely to interact with first and order items on the page accordingly.
+Focusing on content needs can help us create better optimal user experiences–think 
+about which piece of content a user is likely to interact with first and order items 
+on the page accordingly.
+
+Specifying layout content in this way will also ensure that it is interepreted 
+appropriately by screen readers for visually impaired users.
 
-Specifying layout content in this way will also ensure that it is interepreted appropriately by screen readers for visually impaired users.
+The example below has content labelled by its priority. Resize your browser window and 
```

Comment:
```suggestion
The example below has content labeled by its priority. Resize your browser window and 
```
apparently single "l" is more common for american english

Reply:
done